devops/jobs/CloudFlareHitRate.groovy

```diff
@@ -0,0 +1,81 @@
+/*
+
+    Variables without defaults are marked (required)
+
+    Variables consumed for this job:
+        * NOTIFY_ON_FAILURE: (Required - email address)
+        * SECURE_GIT_CREDENTIALS: (Required - jenkins name of git credentials)
+        * ZONE_ID: (Required - The CloudFlare Zone ID)
+        * AUTH_KEY: (Required - Authentication key for the account that would make API calls)
+        * EMAIL: (Required - Email of the account for making API calls)
+
+*/
+package devops.jobs
+
+
+import static org.edx.jenkins.dsl.DevopsTasks.common_parameters
+import static org.edx.jenkins.dsl.Constants.common_wrappers
+
+
+class CloudflareHitRate {
+    public static def job = { dslFactory, extraVars ->
+        assert extraVars.containsKey('ZONE_ID') : "Required ZONE_ID setting missing from configuration"
+        assert extraVars.containsKey('EMAIL') : "Required email(EMAIL) setting missing from configuration"
+        dslFactory.job(extraVars.get("FOLDER_NAME","Monitoring") + "/cloudflare-hit-rate-edx") {
+            wrappers common_wrappers
```

Comment:
Please also include `logRotator common_logrotator`  we don't want to keep the logs forever.